automationScripts/spec.yml

```diff
@@ -3589,6 +3589,68 @@ paths:
       deprecated: false
       security:
         - bearer: []
+  /dialcode/v2/read:
+    post:
+      summary: Get DIAL Code
+      parameters:
+        - name: X-Channel-ID
+          in: header
+          description: >-
+            It the Unique ID to identify the root organization to which the user belongs
```

Comment:
Line 3599: Please add "is" in the description to make the sentence grammatically correct.